src/TingBundle/TingBundle.php

```diff
@@ -28,5 +28,5 @@
 
 class TingBundle extends Bundle
 {
-    const VERSION = '3.2';
+    const VERSION = '3.5';
```

Comment:
I don't know why it is used but couldn't we just deprecate this constant and use [`composer/package-versions-deprecated`](https://packagist.org/packages/composer/package-versions-deprecated) instead ?